website/blog/2021/08/11/interview-TuZhengsong.md

```diff
@@ -1,145 +1,145 @@
 
-“我当时心想，C 语言真的这么难吗？”正松说。
+"I was thinking, "Is C really that hard?" Zhengsong said.
 
-然后他就一直看一直看，看着看着就突然发现编程好像有那么点意思，他看懂了！最后他在毕业前顺利通过了考试。在毕业之后，找到了一份做程序员的工作。
+Then he kept reading and reading and reading, and suddenly he realized that programming seemed to have some meaning, and he understood it! He finally passed the exam before graduating. After graduation, he found a job as a programmer.
 
-“我本来都准备毕业回家找个厂上班了。”，他半开玩笑地说。
+"I was ready to graduate and go home and find a factory to work." , he said half-jokingly.
 
-彼时，他可能不会想到如今的他会成为数个知名开源项目的 developer，参与一些支撑着千万级并发流量的基础设施的开发。
+At that time, he probably wouldn't have thought that today he would be the developer of several well-known open source projects, and involved in the development of some infrastructure that supports millions of concurrent traffic.
 
-## 出象牙塔记
+## Out of the Ivory Tower
 
-> 在工业界，真实的业务场景和学校里学的很不一样的
+> In industry, the real business scenario is very different from what you learn in school.
 
-正松的第一份工作就职于一家做穿戴设备的创业公司，从事智能手环的研发工作。正松和他的前同事们需要通过处理从智能手环上的获取的各种传感器数据，来完成对用户的心率、运动量、血压等的监测。
+Zhengsong's first job was at a wearable device startup, working on the development of smart bracelets. Zhengsong and his former colleagues had to process various sensor data from the smart bracelet to monitor the user's heart rate, exercise, blood pressure and so on.
 
-“工作蛮好玩的，因为在工业界，真实的业务场景和学校里学的很不一样的，很多东西都得从头开始学”，正松说。
+"It was fun because in industry, the real business scenario is very different from what you learn in school, and a lot of things have to be learned from scratch," Masatsu said.
 
-有一天下班之后，这家公司的老板娘突然发了一条信息，说公司的员工暂时休假，什么时候上班等通知。
+One day after work, the owner of this company suddenly sent a message saying that the company's employees were on temporary leave and that they would wait to be notified when to work.
 
-正松当时就觉得不对劲，怎么好好的上着班，突然就休假了？然后他立马打开招聘软件，开始找新的工作。后来才知道，公司的老板跟供应商有合同纠纷，公司短暂停摆了。
+Masatsu felt something was wrong at that time, how to work well, suddenly on vacation? Then he immediately opened the recruitment software and started looking for a new job. Later, he found out that the owner of the company had a contract dispute with the supplier, and the company was briefly shut down.
 
-然后，正松便到另外一家公司从事后端业务开发的工作，学习业界规范、进一步增进技术。过了快一年，因为正松比较喜欢 Nginx、OpenResty、Netty 这类简洁和高性能的技术，他进入一家大公司的开发团队，从事中间件相关开发工作。
+Then, Masatsu went to another company to engage in back-end business development work, to learn the industry norms and further improve technology. After almost a year, he joined the development team of a large company to work on middleware-related development because he liked the simplicity and high performance of technologies like Nginx, OpenResty, and Netty.
```

Comment:
```suggestion
Then, Zhengsong went to another company to engage in back-end business development work, to learn the industry norms and further improve technology. After almost a year, he joined the development team of a large company to work on middleware-related development because he liked the simplicity and high performance of technologies like Nginx, OpenResty, and Netty.
```